pkg/replication/replication.go

```diff
@@ -348,13 +388,6 @@ func (r *replication) ensureHealthy(
 				nodeLogger.WithFields(logrus.Fields{"check": id, "health": status}).Infoln("Node is not healthy")
 			} else {
 				r.logger.WithField("node", node).Infoln("Node is current and healthy")
```

Comment:
@KoishiKomeiji: can you take another look at `ensureHealthy()`?  In particular, when a node has been successfully replicated, how does this method end?